draft-ietf-acme-acme.md

```diff
@@ -1448,7 +1448,6 @@ Location: https://example.com/acme/order/asdf
   "status": "pending",
   "expires": "2016-01-01T00:00:00Z",
 
-  "csr": "jcRf4uXra7FGYW5ZMewvV...rhlnznwy8YbpMGqwidEXfE",
```

Comment:
There should be `identifiers` here, because this is an "object reflecting the client's request"

Reply:
Good catch! You're right. I fixed this in 189d25e and also made sure the number of identifiers present matches the number of authorizations as well.